README.md

```diff
@@ -217,3 +217,21 @@ import { outList } from "rsql-builder";
 
 const op = outList(300, "Taran*", "John Travolta"); // '=out=(300,Taran*,"John Travolta")'
 ```
+
+### Custom operators
+
+New operators can be easily created as follows:
+
+```ts
+import { Operation } from 'rsql-builder';
```

Comment:
Pls, use double quotes instead of single ones. I'll set up linting for MD-files in future.

Reply:
Sure, I didn't realize that double quotes are used in the readme.
Is there a reason for using double quotes in the readme and single quotes everywhere else?

Reply:
That's my bad. In general, I prefer single quotes. But that time, I suppose to be in a hurry preparing release and forget to update README. Most likely, I'll formatted README with single quotes.